saucedemo_framework/utilities/driver_factory.py

```diff
@@ -0,0 +1,23 @@
+from selenium.webdriver import Chrome, Firefox, Edge
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.service import Service
```

Comment:
you import only chrome Service and use it for all other browsers

Reply:
Imported other Services and Options